test/new_relic/slack_notifications_tests/slack_notifier_tests.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require_relative '../../../.github/workflows/scripts/slack_notifications/slack_notifier'
+
+class SlackNotifications < Minitest::Test
+  def test_send_slack_message_zero_args
+    SlackNotifier.stub(:sleep, nil) do
+      assert_raises(ArgumentError) { SlackNotifier.send_slack_message() }
+    end
+  end
+
+  def test_send_slack_message_too_many_args
+    SlackNotifier.stub(:sleep, nil) do
+      assert_raises(ArgumentError) { SlackNotifier.send_slack_message("I am a notification message!", "But I'm one too many") }
```

Comment:
For all NEW code, please stick to a maximum of 120 characters in length.

Reply:
Hehe [yes done](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/commits/6b044062a798dca30a0c6649b73684c31246f934#diff-30d914ac7129feb2d13bf43e052cf29fe91564381ac3623355ac7f02ae77d096R18-R20). I saw it was over a little but thought maybe that little bit that didn't matter so much. Looks better now and noted for the future 😄